node/bundler.ts

```diff
@@ -62,7 +62,7 @@ const bundle = async (
     onAfterDownload,
     onBeforeDownload,
   }
-  const internalFunctionsPath = internalSrcFolder && resolve(internalSrcFolder)
```

Comment:
build does already send an absolute path so not sure why we would need this.

Reply:
I think it was more as a failsafe?